test/integration/scheduler/framework_test.go

```diff
@@ -1333,13 +1336,16 @@ func TestPostBindPlugin(t *testing.T) {
 				if err = testutils.WaitForPodToSchedule(testCtx.ClientSet, pod); err != nil {
 					t.Errorf("Expected the pod to be scheduled. error: %v", err)
 				}
+				select {
+				case <-postBindPlugin.pluginInvokeEventChan:
+				case <-time.After(time.Second * 15):
+					t.Errorf("pluginInvokeEventChan timed out")
+				}
 				if postBindPlugin.numPostBindCalled == 0 {
```

Comment:
Emm...this check should be useless? since if postBindPlugin is called, the `pluginInvokeEventChan` will return, if the postBindPlugin isn't called, the timeout will return.

Reply:
> if the postBindPlugin isn't called, the timeout will return.

`t.Errorf` won't return - the logic proceeds to L1344.

Reply:
> > if the postBindPlugin isn't called, the timeout will return.
> 
> `t.Errorf` won't return - the logic proceeds to L1344.

Yea, I mean only the t.Errorf at line 1342 is enough, line 1342 already ensure the plugin isn't called.

Reply:
I see your point. Although the root cause of L1342 and L1344 ~ L1346 is the same, but their symptoms are different, so IMO keeping L1344 ~ L1346 as is can make the semantics more explicit.

Reply:
got it, thanks